src/core/recv_buffer.c

```diff
@@ -321,7 +322,7 @@ QuicRecvBufferResize(
                     Span - LengthTillWrap);
             }
             RecvBuffer->ReadStart = 0;
-            RecvBuffer->Capacity = TargetBufferLength;
+            RecvBuffer->Capacity = NewChunk->AllocLength;
```

Comment:
I wonder if it's worth a debug assert to say these are the same/interchangeable?